CHANGELOG.md

```diff
@@ -14,8 +14,8 @@
 * Add configuration option `WithSamlAdfs` to `NewVCDClient()` to support SAML authentication using
   Active Directory Federations Services (ADFS) as IdP using WS-TRUST auth endpoint
   "/adfs/services/trust/13/usernamemixed"
-  [#304](https://github.com/vmware/go-vcloud-director/pull/304)
-  
+  [#304](https://github.com/vmware/go-vcloud-director/pull/304)  
+* Added methods `vapp.UpdateNetworkFirewallRules`, `vapp.UpdateNetworkFirewallRulesAsync`, `vapp.GetVappNetworkById`, `vapp.GetVappNetworkByName` and `vapp.GetVappNetworkByNameOrId` [#308](https://github.com/vmware/go-vcloud-director/pull/308)
```

Comment:
Just for better understanding - is there not such thing as `POST` (create) at all. I see Update uses `PUT`.

Reply:
Api has only PUT, GET